portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="script.js"></script>
+  </head>
+   <body>
+        <h1>About Me</h1>
+        <div class="row">
+            <div class="column left">
+                <img class="profile-picture" src="images/profileimage.jpg" width="250"/>
+                <br>
+                <br>
+                <br>
+                <img src="images/nsbeimage.png" width="300"/>
+                <img src="images/educationimage.jpg" width="300"/>
+            </div>
+            <div class="column right">
+                <p class="bio">
+                    Hello, my name is Kiera Brown, and I'm studying computer science at Howard University.
+                    I'm very passionate about devloping tech to help the community. 
+                    Some of my hobbies include sleeping, dancing, cook, and traveling.
+                </p>
+                <p>
+                    Broadening the tech world to underrepresented groups has always been something I’m passionate about. Accessibility and inclusion are extremely important. 
+                    Since I first discovered that I wanted to pursue a career tech, I knew that I would be working in an environment where both women and African Americans were underrepresented, and I wanted to do something to change that. 
```

Comment:
nit: typo - "career in tech" instead of "career tech"

Reply:
Fixed